src/supernode/requests/debug.cpp

```diff
@@ -119,13 +119,26 @@ Status doAnnounce(const Router::vars_t& vars, const graft::Input& input,
     return sendAnnounce(vars, input, ctx, output);
 }
 
+
+Status closeStakeWallets(const Router::vars_t& vars, const graft::Input& input,
+                        graft::Context& ctx, graft::Output& output)
+{
+    FullSupernodeListPtr fsl = ctx.global.get(CONTEXT_KEY_FULLSUPERNODELIST, FullSupernodeListPtr());
+    auto items = fsl->items();
+    for (const auto &sn : items) {
+        fsl->remove(sn);
+    }
+    return Status::Ok;
+}
+
 void __registerDebugRequests(Router &router)
 {
 #define _HANDLER(h) {nullptr, graft::supernode::request::debug::h, nullptr}
     // /debug/supernode_list/0 -> do not include inactive items
     // /debug/supernode_list/1 -> include inactive items
     router.addRoute("/debug/supernode_list/{all:[0-1]}", METHOD_GET, _HANDLER(getSupernodeList));
     router.addRoute("/debug/announce", METHOD_POST, _HANDLER(doAnnounce));
+    router.addRoute("/debug/close/", METHOD_POST, _HANDLER(closeStakeWallets));
```

Comment:
'close_wallets'?

Reply:
thanks, sounds better, renamed to "close_wallets"